### DISCUSSION TITLE
Не запускается проект

### DISCUSSION THREAD

**Discussion by SaBond0** [AvaloniaUI/Avalonia#14205] [Q&A]
недавно скачал Avalonia, пытаюсь запустить проект, в сборке пишет, всё успешно. но по факту вылезает сообщение: "Проект библиотека классов которого имеет "Тип выходных данных", нельзя запустит на прямую". я так понимаю где то нужно прописать run такое-то окно, но пока не очень понимаю где и как прописать, помогите пожалуйста

**Comment by maxkatz6** [comment]
You are probably running a library project instead of executable project.

**Reply by SaBond0** [reply]
как возможно изменить ?

**Reply by thevortexcloud** [reply]
It will be in your project file. Otherwise just use the default templates. EG `dotnet new avalonia.mvvm`

**Reply by SaBond0** [reply]
Вроде получилось, но цвета не совсем совпадают, типо в боксе текст - черного,а фон белого, а когда работает все наоборот

**Reply by thevortexcloud** [reply]
Google translate is not working too well for this one. But it sounds like it's a theme problem? Can you show your `app.axaml` file?

**Reply by SaBond0** [reply]
![image](https://github.com/AvaloniaUI/Avalonia/assets/156509288/01ccdcf1-a8f9-44ae-b82e-f076ed9daf22)

**Reply by maxkatz6** [reply]
"App.axaml" file can't be previewed.

**Reply by SaBond0** [reply]
what do you mean

**Reply by thevortexcloud** [reply]
This looks alright to me. Is the problem what the previewer is showing or when you run the app?

**Reply by SaBond0** [reply]
run the app

**Reply by SaBond0** [reply]
![image](https://github.com/AvaloniaUI/Avalonia/assets/156509288/ca06306b-917f-4950-b9d5-38a579a1b829)
![image](https://github.com/AvaloniaUI/Avalonia/assets/156509288/90d3ca40-2748-494e-b4e1-023a23528483)

**Reply by thevortexcloud** [reply]
Ah, the problem is the previewer theme is not matching the runtime theme? By default Avalonia will default to what ever your system theme is. You can override that in `App.axaml` by changing `RequestedThemeVariant`

**Reply by SaBond0** [reply]
I understand what you mean, but I don't need it, because I want my form to open separately from the console

**Reply by thevortexcloud** [reply]
Can you please clarify what you mean by "console"? I am starting to think it means something else other than the .NET console.

**Comment by maxkatz6** [comment]
Please visit http://t.me/AvaloniaRU for quick questions in russian.

**Reply by SaBond0** [reply]
i will try write on English, my Yandex translate +- good working
